platform/core/src/classes/metadata/StudyMetadata.js

```diff
@@ -186,9 +196,65 @@ export class StudyMetadata extends Metadata {
   }
 
   /**
-   * Creates a set of series to be placed in the Study Metadata
-   * The series that appear in the Study Metadata must represent
-   * imaging modalities.
+   * Adds the displaySets to the studies list of derived displaySets.
+   * @param {object} displaySet The displaySet to append to the derived displaysets list.
+   */
+  _addDerivedDisplaySet(displaySet) {
+    this._derivedDisplaySets.push(displaySet);
+    // --> Perhaps that logic should exist in the extension sop class handler and this be a dumb list.
+    // TODO -> Get x Modality by referencedSeriesInstanceUid, FoR, etc.
+  }
+
+  /**
+   * Returns a list of derived datasets in the study, filtered by the given filter.
+   * @param {object} filter An object containing search filters with
+   * optional fields for the modality, referencedSeriesInstanceUID and referencedFrameOfReferenceUID.
+   */
+  getDerivedDatasets(filter) {
```

Comment:
List of supported filter options would be nice; either deconstructed in signature, or listed in JSDoc params

Reply:
Filter options added in jsdocs 👍